app_dart/lib/src/service/config.dart

```diff
@@ -59,6 +63,8 @@ interface class Config {
   final CacheService _cache;
   final SecretManager _secrets;
 
+  DynamicConfig dynamicConfig;
```

Comment:
Can we make this private? I wouldn't want to see anyone _but_ `Config` read from this directly.

Reply:
Talked off line - done by pulling the updater into here so it has access.